templates/index.html

```diff
@@ -0,0 +1,58 @@
+{%- set base_url = '/'|url(alt='es') -%}
```

Comment:
what is alt='es' needed for  ?

Reply:
I defined 2 languages in the config file: https://github.com/PyConColombia/website-2021/pull/2/files#diff-42b38e342f7d21f387355ed2dfd23b61R9

So, it's necessary choose the default language in all templates.

Reference: https://www.getlektor.com/docs/content/alts/